contracts/payment/PaymentSplitter.sol

```diff
@@ -22,7 +22,11 @@ contract PaymentSplitter {
     address[] private _payees;
 
     /**
-     * @dev Constructor.
+     * @dev Creates an instance of `PaymentSplitter` where each account in `payees` is assigned the number of shares at
+     * the matching position in the `shares` array. Each account will then receive payments proportional to the
+     * percentage of the shares that they were assigned.
```

Comment:
>Each account will then receive payments proportional to the percentage of the shares that they were assigned.

This wording makes it sound like that will happen during construction. IMO this general behavior information belongs at the top declaration, and we should only describe how each part of the contract plays a role in said behavior. e.g.:

>Funds send to this contract will be split among `payees` proportionally, according to the percentage of the `shares` that they were assigned.

Reply:
Good point, I forgot to edit the contract docs.